src/mock-data/authors.js

```diff
@@ -1,22 +1,26 @@
-import author1 from './author1.json'
-import author2 from './author2.json'
-import author3 from './author3.json'
-import author4 from './author4.json'
-import authors from './authors.json'
+import author from './author.json'
+import authorPost from './author-post.json'
 
 // lodash
+import assign from 'lodash/assign'
 import find from 'lodash/find'
+import map from 'lodash/map'
 
 const _ = {
+  assign,
+  map,
   find,
 }
 
+const posts = Array(50).fill(authorPost)
+
 const mocks = {
-  authors: authors.data,
-  author1: author1.data,
-  author2: author2.data,
-  author3: author3.data,
-  author4: author4.data,
+  authors: Array(100).fill(author),
+  authorPosts: _.map(posts, (post, index) => {
+    return _.assign({}, post, {
+      id: index,
```

Comment:
Why do we have to redefine the id of the post?

Reply:
To make sure the mock data can be represented in pagination in order to check UI behavior, I mocked different post ids since they are placed in `entitiesForAuthors` substate in redux store by id.

Reply:
My concern would be that if the id is replaced with page index. Is it still able to retrieve the post through the id?

Reply:
No, it can not. Your concern is valid.
I update slug for mock post in latest commit.